src/spell.py

```diff
@@ -1,78 +1,97 @@
 #!/usr/bin/env python
 from __future__ import print_function
-
-import sys
-from typing import Dict
+import argparse
 
 law_enforcement_spelling = {
-    'a': 'Adam',
-    'b': 'Boston',
-    'c': 'Charles',
-    'd': 'David',
-    'e': 'Edward',
-    'f': 'Frank',
-    'g': 'George',
-    'h': 'Henry',
-    'i': 'Ida',
-    'j': 'James',
-    'k': 'King',
-    'l': 'Lincoln',
-    'm': 'Mary',
-    'n': 'Nora',
-    'o': 'Oscar',
-    'p': 'Paul',
-    'q': 'Queen',
-    'r': 'Robert',
-    's': 'Sam',
-    't': 'Tom',
-    'u': 'Union',
-    'v': 'Victor',
-    'w': 'William',
-    'x': 'X-Ray',
-    'y': 'Young',
-    'z': 'Zebra',
+    "a": "Adam",
+    "b": "Boston",
+    "c": "Charles",
+    "d": "David",
+    "e": "Edward",
+    "f": "Frank",
+    "g": "George",
+    "h": "Henry",
+    "i": "Ida",
+    "j": "James",
+    "k": "King",
+    "l": "Lincoln",
+    "m": "Mary",
+    "n": "Nora",
+    "o": "Oscar",
+    "p": "Paul",
+    "q": "Queen",
+    "r": "Robert",
+    "s": "Sam",
+    "t": "Tom",
+    "u": "Union",
+    "v": "Victor",
+    "w": "William",
+    "x": "X-Ray",
+    "y": "Young",
+    "z": "Zebra",
 }
 
 nato_spelling = {
-    'a': 'Álfa',
-    'b': 'Brávo',
-    'c': 'Chárlie',
-    'd': 'Délta',
-    'e': 'Écho',
-    'f': 'Fóxtrot',
-    'g': 'Gólf',
-    'h': 'Hotél',
-    'i': 'Índia',
-    'j': 'Júliett',
-    'k': 'Kílo',
-    'l': 'Líma',
-    'm': 'Mike',
-    'n': 'Novémber',
-    'o': 'Óscar',
-    'p': 'Papá',
-    'q': 'Quebéc',
-    'r': 'Rómeo',
-    's': 'Siérra',
-    't': 'Tángo',
-    'u': 'Úniform',
-    'v': 'Víctor',
-    'w': 'Whísky',
-    'x': 'X-ray',
-    'y': 'Yánkee',
-    'z': 'Zúlu'
+    "a": "Álfa",
+    "b": "Brávo",
+    "c": "Chárlie",
+    "d": "Délta",
+    "e": "Écho",
+    "f": "Fóxtrot",
+    "g": "Gólf",
+    "h": "Hotél",
+    "i": "Índia",
+    "j": "Júliett",
+    "k": "Kílo",
+    "l": "Líma",
+    "m": "Mike",
+    "n": "Novémber",
+    "o": "Óscar",
+    "p": "Papá",
+    "q": "Quebéc",
+    "r": "Rómeo",
+    "s": "Siérra",
+    "t": "Tángo",
+    "u": "Úniform",
+    "v": "Víctor",
+    "w": "Whísky",
+    "x": "X-ray",
+    "y": "Yánkee",
+    "z": "Zúlu",
 }
 
+parser = argparse.ArgumentParser(
+    description="A simple command line tool "
+    "to display a spelling for any phrase "
+    "using a phonetic alphabet."
+)
+parser.add_argument(
+    "--alphabet",
+    default="nato",
+    choices=["nato", "law_enforcement"],
+    dest="alphabet",
+)
+parser.add_argument(dest="phrase")
```

Comment:
This version doesn't allow user to pass phrases, it only allows one word. I would like to keep the following usage possible:
```
$ spell a long phrase
```
Right now it fails with an error: "error: unrecognized arguments: long phrase"

Reply:
Fixed!
```bash
$ ./src/spell.py spell a long phrase
Siérra
Papá
Écho
Líma
Líma
Álfa
Líma
Óscar
Novémber
Gólf
Papá
Hotél
Rómeo
Álfa
Siérra
Écho
```
I decided to remove spaces and other non-alpha characters from the output because it looked weird with verbose:
` : ` was the output line for any phrases with spaces, for example.
Let me know if you want me to leave spaces as part of the output.

Reply:
I would like to keep spaces, because it helps you to separate words visually. For verbose mode we can process spaces separately and print an empty line.  
Also I think that it's important to keep numbers as well, but I'm not sure how it should work. 

But I think these are issues for separate PRs. For now let's keep things simple and revert the behavior to the old one (if it works with your application).